### PR TITLE
Documentation implemented with docusaurus

### PR DESCRIPTION
**Pull request by eppak**
This is the first version of the documentation based on docusaurus, list of what is done:

- Based on docusaurus in monorepo configuration.
- Updated base resources in static/img with original branding.
- Created a splash page with features list.
- Created a new command to generate changelog.
- Moved all documentation on docs folder.

Things to improve:

- The changelog command use a sed to add a title, can be better implemented.
- Deploy url need to be updated before going in production.
- Adapt colors if needed: used the logo main color for both light/dark, all shades not implemented.
- Need to be integrated with a pipeline when a new version is published:
    - cd website
    - npm install
    - npm run changelog
    - npm run contributors
    - npm run build
    - Publish the "build" folder

### CODE REVIEW

**Comment by Cadienvan** [comment]
@mateonunez I had a look and it seems cool enough, didn't have time to look at it through yet.

Want to give it a look as you have more experience than me on docs and docusaurus in particular?

**Comment by mateonunez** [comment]
I will take a look at this PR as soon as possible. 🙏

File: website/README.md
Comment (mateonunez) [inline website/README.md:1]: Can you replace this with a permalink to the `intro.md` page?

File: website/package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "website",
+  "version": "0.0.0",
+  "private": true,
```
Comment (mateonunez) [inline website/package.json]: Replace with `false`.

File: website/package.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "website",
```
Comment (mateonunez) [inline website/package.json]: I suggest: `@expirables/docs`.

What do you think, @Cadienvan?

File: website/src/components/HomepageFeatures/index.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+type FeatureItem = {
+  title: string;
+  Svg: React.ComponentType<React.ComponentProps<'svg'>>;
+  description: JSX.Element;
+};
+
+const FeatureList: FeatureItem[] = [
+  {
+    title: 'Common structures',
+    Svg: require('@site/static/img/undraw_task_re_wi3v.svg').default,
```
Comment (mateonunez) [inline website/src/components/HomepageFeatures/index.tsx]: I think mixing `import` and `require` is not a good idea for transpiler and bundler...

You can create a `<Svg />` component and then reuse it in other parts, what do you think?

File: website/src/components/HomepageFeatures/index.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      <>
+      Based on MIT license.
+      </>
+    ),
+  },
+];
+
+function Feature({title, Svg, description}: FeatureItem) {
+  return (
+    <div className={clsx('col col--4')}>
```
Comment (mateonunez) [inline website/src/components/HomepageFeatures/index.tsx]: You don't need `clsx` here

File: website/src/pages/markdown-page.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (mateonunez) [inline website/src/pages/markdown-page.md]: Remove this file

**Review by mateonunez — COMMENTED** [review]
Hey, @eppak! This is an amazing job! Thank you so much! 🙏

I think we can improve some small things, but this is a great starting point.

I just left you a couple of comments that we can discuss.

Also, how about moving `website` to the `packages/*` folder? 

```bash
expirables/packages/website
```

That way we can prepare this repo to hold other packages.

File: website/package.json (hunk 2 above)
Comment (Cadienvan) [reply website/package.json]: I agree

File: website/README.md
Comment (eppak) [reply website/README.md:1]: Here I ask you some more specification, I understand it generally but not as a github related topic

File: website/package.json (hunk 1 above)
Comment (eppak) [reply website/package.json]: Done, set version to 0.0.1

File: website/package.json (hunk 2 above)
Comment (eppak) [reply website/package.json]: Done.

File: website/src/components/HomepageFeatures/index.tsx (hunk 3 above)
Comment (eppak) [reply website/src/components/HomepageFeatures/index.tsx]: This is the default code of docusaurus, can you share a link to this problem just to study it, thanks. Maybe was a better idea to start directly with ts.

Done, moved to a direct link using img.

File: website/src/components/HomepageFeatures/index.tsx (hunk 4 above)
Comment (eppak) [reply website/src/components/HomepageFeatures/index.tsx]: Done.

File: website/src/pages/markdown-page.md (hunk 5 above)
Comment (eppak) [reply website/src/pages/markdown-page.md]: Removed

File: website/src/components/HomepageFeatures/index.tsx (hunk 3 above)
Comment (Cadienvan) [reply website/src/components/HomepageFeatures/index.tsx]: @mateonunez I guess we could leave it as it is as of right now, what do you think?

**Comment by Cadienvan** [comment]
@mateonunez wanna bring it in then?

**Comment by mateonunez** [comment]
Hey folks, let's see what CI says and then I can approve and merge.

Amazing job, @eppak! Thank you so much 🙏

**Review by mateonunez — APPROVED** [review]
LGTM! 🤗

**Pull request merged**